tests/src/Simple/HelloWasm/Program.cs

```diff
@@ -159,6 +159,10 @@ private static unsafe int Main(string[] args)
         arrayTest[1].Value = "Array load/store test: Ok.";
         PrintLine(arrayTest[1].Value);
 
+        int ii = 0;
```

Comment:
As I mentioned on the other thread, please add ternary operator tests to cover the basic block entry stack cases